prep-javascript-functions/main.js

```diff
@@ -0,0 +1,27 @@
+// function addTwoNumbers(x, y) {
+//   return x + y;
+// }
+
+// var sum = addTwoNumbers(2, 2);
+// console.log(sum);
+
+// function convertHoursToMinutes(hour) {
+//   return hour * 60;
+// }
+
+// var inMinutes = convertHoursToMinutes(2);
+// console.log(inMinutes);
+
+// function getGreeting(name) {
+//   return 'Hello' + ' ' + name + '!';
+// }
+
+// var greeting = getGreeting('World');
+// console.log(greeting);
```

Comment:
Please uncomment these and leave in the console logs. Also, there are a few functions missing.